packages/safety-suite-api/src/api/dashboardSink/fleetPerfomanceDashboard.ts

```diff
@@ -0,0 +1,335 @@
+import {Request} from '@idelic/safety-net';
+
+import {runApi} from '../../runApi';
+import {ApiOptions, ApiSuccessResponse} from '../../types';
+
+interface Sort {
+  column: string;
+  direction?: string;
+}
+
+interface Query {
+  customerAlias: string;
+  groupIds?: string[];
+}
+
+interface TableQuery extends Query {
+  sort: Sort;
+  page?: number;
+  size?: number;
+}
+
+interface Model {
```

Comment:
Is `Model` a good name to use here based on how much in our system is currently called `Model`?  Mind you, I don't really have better ideas!  Maybe... `BaseColumns`?  `BaseFields`?  🤷‍♀️